pkg/apis/build/v1alpha2/build.go

```diff
@@ -141,6 +141,13 @@ func (b *Build) NotaryV1Config() *corev1alpha1.NotaryV1Config {
 	return b.Spec.Notary.V1
 }
 
+func (b *Build) DefaultProcess() string {
+	if b.Spec.DefaultProcess == "" {
```

Comment:
I see a few other methods that have `b == nil` checks making me think that there are cases where a `nil` Build object is used and may be called. Should this method also have the same check?

FWIW, I only bring it up because of the existing pattern. In reality it would be more ideal that a `nil` Build object isn't called at all.

Reply:
Makes sense, added nil check to be safe.

Reply:
FWIW. The nil check is not necessary here because DefaultProcess will always be called on an existing build.